xline/src/server/xline_server.rs

```diff
@@ -91,25 +95,30 @@ where
             lease_cmd_rx,
             Arc::clone(&state),
             Arc::clone(&header_gen),
-            storage.clone(),
-        ));
+            Arc::clone(&storage),
+        )?);
         let kv_storage = Arc::new(KvStore::new(
             lease_cmd_tx.clone(),
             del_rx,
             Arc::clone(&header_gen),
-            storage.clone(), // TODO: implement bucket storage
-        ));
-        let auth_storage = Arc::new(AuthStore::new(lease_cmd_tx, key_pair, header_gen, storage));
+            Arc::clone(&storage), // TODO: implement bucket storage
```

Comment:
remove this todo, bucket in etcd is our table

Reply:
Fixed